app/lib/applications_controller_methods.rb

```diff
@@ -52,8 +52,7 @@ def find_applications
   end
 
   def find_cinstance
-    @cinstance = accessible_not_bought_cinstances.includes(plan: %i[service original plan_metrics pricing_rules])
-                                                 .find(params[:id])
+    @cinstance = accessible_not_bought_cinstances.find(params[:id])
```

Comment:
Assuming you never reference these associations later (service, original, plan_metrics, pricing_rules)?  Bc if you do, the includes will save you additional queries to get those objects.